src/app/features/fee-editor/fee-editor.context.ts

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext } from 'react';
+
+import type { MarketData, Money } from '@leather.io/models';
+
+export type EditorFeeType = 'slow' | 'standard' | 'fast' | 'custom';
+
+export interface EditorFee {
+  type: EditorFeeType;
+  feeRate: number | null;
+  feeValue: Money | null;
+  time: string;
+}
```

Comment:
`FeeEditor` is a great name for this component. Calling fee data within it `EditorFee` I find a bit hard to follow.

`FeeSelection`, `SelectedFee`, `FeeEdtiorSelection`?